src/main/java/frc/robot/Subsystems/ClimberPlacer.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.Subsystems;
+
+import com.spikes2212.command.genericsubsystem.MotoredGenericSubsystem;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import edu.wpi.first.wpilibj.DigitalInput;
+import frc.robot.RobotMap;
+
+/**
+ * This is a motor controlled subsystem of the placer of the climber. which climbs up to the third bar from the second
```

Comment:
Describe this subsystem better (For example: "Controls the position of the climber's bars")